protocol/src/common/protocol.$.ts

```diff
@@ -15,12 +15,16 @@ import { WorkDoneProgressBegin, WorkDoneProgressEnd, WorkDoneProgressReport } fr
 
 // @ts-ignore 6196
 namespace SetTraceNotification {
-	export const type = new ProtocolNotificationType<SetTraceParams, void>('$/setTrace');
+  export const method: '$/setTrace' = '$/setTrace';
```

Comment:
@michaelpj these indents still seem to be off. Looks like that use 2 spaces, but in this repository we indent using tabs. Could you please address this as well. 

Thanks.

Reply:
Argh sorry, I thought I'd got all of them >:(